lib/cli/driver.js

```diff
@@ -9,6 +9,9 @@ import NPM from './npm';
 import { errAndQuit, log, spinWith, JSON_SPACES } from './utils';
 import { util, fs } from 'appium-support';
 
+const UPDATE_ALL = 'installed';
```

Comment:
should we have documented somewhere the set of rules for drivers naming and which of these names are reserved?

Reply:
yeah, this should be in the documentation. i don't see this as dangerous though, if someone creates a driver called 'installed' it will just never be updatable using this command. would be a dumb move though.

also i don' tmind if people create drivers with the same name, just whichever one is installed first will be the one that's used, and others won't be able to be installed.